augur/filter/subsample.py

```diff
@@ -368,6 +368,12 @@ def get_weighted_group_sizes(groups, group_by, weights_file, target_total_size,
     rng = np.random.default_rng(random_seed)
     weights[SIZE_COLUMN] = (weights[FRACTIONAL_SIZE_COLUMN].add(rng.random(len(weights)))).astype(int)
 
+    # TODO: consider adding an option to output the computed sizes since that
```

Comment:
Added in 2ee8b26fa539b6a9fff17a8e53e6a0866ccd2e73 since it's useful info for the [other thread](https://github.com/nextstrain/augur/pull/1454#discussion_r1596214631).

Reply:
Leaving this open to add **number of outputted sequences**. I don't think it's gonna be trivial to add (properly), but need to dig a little deeper.

Reply:
This was implemented for https://github.com/nextstrain/augur/pull/1454#discussion_r1708166896